mypy.ini

```diff
@@ -3,32 +3,41 @@ packages = baybe
 
 ; at some point, these excludes should all be gone ...
 exclude = (?x)(
-          baybe/constraints
-          | baybe/recommenders
+          baybe/recommenders
           | baybe/searchspace
           | baybe/serialization
           | baybe/strategies
           | baybe/surrogates
-          | baybe/utils/basic.py
-          | baybe/utils/boolean.py
-          | baybe/utils/botorch_wrapper.py
-          | baybe/utils/chemistry.py
           | baybe/utils/dataframe.py
-          | baybe/utils/numeric.py
-          | baybe/utils/sampling_algorithms.py
           | baybe/acquisition.py
-          | baybe/campaign.py
           | baybe/deprecation.py
           | baybe/exceptions.py
           | baybe/objective.py
           | baybe/scaler.py
           | baybe/simulation.py
           | baybe/surrogate.py
-          | baybe/telemetry.py
           )
 
+[mypy-botorch.test_functions]
```

Comment:
Can you elaborate what was the problem with the stubs here?

Reply:
that is an `excellent` question and no, I cant :/

this was one of two mypy errors that kept appearing during this development locally for me. At the time of publishing the PR they were consistently gone.

The the CI failed and showed me this again, hence I added this exclusion. The other error was even more random, but so far hasnt appeared again.

I dont know why there would be an issue witht eh botorch typing here and did not investigate, but it was the plain warning about missing typing from a package

Reply:
Ok, I'm fine with having it excluded if the stubs are not working / missing. Minor dependency anyway ...